controllers/users.js

```diff
@@ -33,7 +35,7 @@ router.post(
   '/',
   handleRouteErrors(async (req, res) => {
     const user = await userService.register(req.body.email, req.body.password);
-    res.send(user);
+    res.send(console.log(user));
```

Comment:
console.log(user) returns undefined. why are you sending undefined to response? get rid of console.logs in your code. if you want to log something permanently pls use logger